core/App/json_utils.h

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (C) 2021-2022 Intel Corporation
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ *   1. Redistributions of source code must retain the above copyright
+ *      notice, this list of conditions and the following disclaimer.
+ *   2. Redistributions in binary form must reproduce the above copyright
+ *      notice, this list of conditions and the following disclaimer in
+ *      the documentation and/or other materials provided with the
+ *      distribution.
+ *   3. Neither the name of Intel Corporation nor the names of its
+ *      contributors may be used to endorse or promote products derived
+ *      from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+#ifndef _JSON_UTILS_H
+#define _JSON_UTILS_H
+
+#include <jsoncpp/json/json.h>
+
+static char* StringToChar(std::string str)
+{
+    char *retChar = NULL;
+    if (str.size() > 0) {
+        int len = str.size() + 1;
+        retChar = (char *)malloc(len * sizeof(uint8_t));
+        if(retChar != nullptr){
+            memset(retChar, 0, len);
+            memcpy(retChar, str.c_str(), len);
+        }
+    }
+    return retChar;
+}
+
+class RetJsonObj{
+public:
+    const int CODE_SUCCESS = 200;
+    const int CODE_BAD_REQUEST = 400;
+    const int CODE_FAILED = 500;
+

```

Comment:
private

Reply:
And could align the naming like m_xxx, m_yyy for the private member var?

Reply:
It will be fix